auth-mailchimp-sync/functions/src/index.ts

```diff
@@ -95,5 +99,83 @@ export const removeUserFromList = functions.handler.auth.user.onDelete(
     } catch (err) {
       logs.errorRemoveUser(err);
     }
-  }
-);
+  });
+
+export const addExistingUsersToList = functions.tasks
+  .taskQueue()
+  .onDispatch(async (data) => {
+    const runtime = getExtensions().runtime();
+    if (!config.doBackfill) {
+      await runtime.setProcessingState(
+        "PROCESSING_COMPLETE",
+        "No existing documents imported into BigQuery."
+      );
+      return;
+    }
+    const nextPageToken = data.nextPageToken;
+    const pastSuccessCount = (data["successCount"] as number) ?? 0;
+    const pastErrorCount = (data["errorCount"] as number) ?? 0;
+
+    const res = await getAuth().listUsers(100, nextPageToken);
```

Comment:
Since this is going to be a sample, let's add a comment above this explaining the choice of 100. (i.e. "We are sure we can complete 100 users within the 9 minute limit" or something like that).

Reply:
Will do - note that the real reason I chose 100 is 'I arbitrarily made it up. Need to actually verify that it is a reasonable choice.